src/main/java/net/ccbluex/liquidbounce/features/module/modules/render/ClickGUI.kt

```diff
@@ -33,9 +33,12 @@ object ClickGUI : Module("ClickGUI", ModuleCategory.RENDER, keyBind = Keyboard.K
     val maxElementsValue = IntegerValue("MaxElements", 15, 1, 30)
     val fadeSpeedValue = FloatValue("FadeSpeed", 1f, 0.5f, 2f)
     val scrollsValue = BoolValue("Scrolls", false)
+    val spacedModulesValue = BoolValue("SpacedModules", true)
```

Comment:
Put the value at "false", to keep the original design.

Reply:
Or it could be kept like this to make feel LB a bit newer.

(I don't think many people would notice this since who really looks into ClickGUI or ArrayList settings every day?)

Reply:
> Or it could be kept like this to make feel LB a bit newer.
> 
> (I don't think many people would notice this since who really looks into ClickGUI or ArrayList settings every day?)

Doesn't really feel newer, makes it feel worse. This option should be used only in custom HUDs...